solutions/src/main/java/ru/sberbank/school/task06/CollectionUtils.java

```diff
@@ -0,0 +1,175 @@
+package ru.sberbank.school.task06;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class CollectionUtils {
+
+    /**
+     * Добавление всех элементов одной коллекции в другую.
+     *
+     * @param source      источник
+     * @param destination потребитель
+     */
+    public static <T> void addAll(List<? extends T> source, List<? super T> destination) {
+        destination.addAll(source);
+    }
+
+    /**
+     * Инициализация листа с реализаций java.uXtil.ArrayList.
+     *
+     * @return пустой ArrayList
+     */
+    public static <T> List  newArrayList() {
+        return new ArrayList<T>();
+    }
+
+    /**
+     * Поиск индекса элемента в списке.
+     *
+     * @param source список, в котором искать элемент
+     * @param o      элемент
+     * @return индекс элемента
+     */
+    public static <T> int indexOf(List<? extends T> source, T o) {
+        return source.indexOf(o);
+    }
+
+    /**
+     * Обрезание списка заданным числом элементов. limit({1, 2 ,3}, 2) => {1, 2}
+     *
+     * @param source список
+     * @param size   необходимое число элементов
+     * @return сокращенный список
+     */
+    public static <T> List limit(List<? extends T> source, int size) {
+        return source.subList(0, size);
+    }
+
+    /**
+     * Добавление элемента в коллекцию.
+     *
+     * @param dest коллекция, в которую необходимо вставить элемент
+     * @param o    элемент
+     */
+    public static <T> void add(List<? super T > dest, T o) {
+        dest.add(o);
+    }
+
+
+    /**
+     * Удаление всеъх элементов одной коллекии из другой.
+     *
+     * @param removeFrom коллекция, из которой удалять элементы
+     * @param toRemove   коллекция, в которой лежат элементы для удаления из коллекции removeFrom
+     */
+    public static <T> void removeAll(List<? super T> removeFrom, List<? extends T> toRemove) {
+            removeFrom.removeAll(toRemove);
```

Comment:
&#39;method def&#39; child has incorrect indentation level 12, expected level should be 8.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1527796490>*